docs/CHANGELOG.asciidoc

```diff
@@ -40,10 +41,21 @@ This section summarizes the changes in each release.
 * <<release-notes-7.0.0-alpha1>>
 
 --
+[[release-notes-7.9.1]]
+== {kib} 7.9.1
+
+See also <<breaking-changes-7.9,breaking changes in 7.9>>.
+
+[float]
+[[bug-v7.9.1]]
+=== Bug fix
+Dashboard:: 
+* OriginatingApp Connection not lost after Create New {kibana-pull}74400[#74400]
```

Comment:
This link points to the issue.  Should link to the PR:  https://github.com/elastic/kibana/pull/74420